Doc/library/pathlib.rst

```diff
@@ -946,6 +946,111 @@ call fails (for example because the path doesn't exist).
    to the directory after creating the iterator, whether a path object for
    that file be included is unspecified.
 
+.. method:: Path.walk(top_down=True, on_error=None, follow_symlinks=False)
+
+   Generate the file names in a directory tree by walking the tree
+   either top-down or bottom-up.
+
+   For each directory in the directory tree rooted at *self* (including
+   *self* but excluding '.' and '..'), yields a 3-tuple
+   ``(dirpath, dirnames, filenames)``
+
+   *dirpath* is a Path to the directory, *dirnames* is a list of the names
+   of the subdirectories in *dirpath* (excluding ``'.'`` and ``'..'``), and
+   *filenames* is a list of the names of the non-directory files in *dirpath*.
+   Note that the names in the lists contain no path components. To get a full
+   path (which begins with *self*) to a file or directory in *dirpath*, do
+   ``dirpath / name``. Whether or not the lists are sorted depends on the file
+   system. If a file or a directory is removed from or added to the *dirpath*
+   during the generation of *dirnames* and *filenames*, it is uncertain whether
+   the new entry will appear in the generated lists.
```

Comment:
I don't follow this sentence. Would it not apply to `glob()`, `iterdir()` etc too? If so, why call it out here specifically?